apps/gkyl_gyrokinetic_priv.h

```diff
@@ -1643,6 +1764,107 @@ void gk_neut_species_moment_calc(const struct gk_species_moment *sm,
 void gk_neut_species_moment_release(const struct gkyl_gyrokinetic_app *app,
   const struct gk_species_moment *sm);
 
+/** gk_neut_species_lte API */
+
+/**
+ * Initialize species lte object.
+ *
+ * @param app Gyrokinetic app object
+ * @param s Neutral species object 
+ * @param lte Neutral species lte object
+ * @param corr_inp Input struct with moment correction inputs
+ */
+void gk_neut_species_lte_init(struct gkyl_gyrokinetic_app *app, struct gk_neut_species *s,
```

Comment:
same as my comment for gk_species_lte above: we are not going to be using Maxwell-Juttner distributions here, so let's just call this _maxwellian instead of _lte. Hence rename
- gk_neut_species_lte_init -> gk_neut_species_maxwellian_init
- gk_neut_species_lte_from_moms -> gk_neut_species_maxwellian_from_moms
- gk_neut_species_lte -> gk_neut_species_maxwellian_from_distf
- gk_neut_species_lte_write_max_corr_status -> gk_neut_species_maxwellian_write_corr_status
- gk_neut_species_lte_release -> gk_neut_species_maxwellian_release